tests/e2e/e2e_suite_test.go

```diff
@@ -93,6 +104,8 @@ var runTimeClientForSuiteRun client.Client
 var veleroClientForSuiteRun veleroClientset.Interface
 var csiClientForSuiteRun *snapshotv1client.Clientset
 var dpaCR *DpaCustomResource
+var knownFlake bool
```

Comment:
since we are not adding this to must-gather tests, only backup/restore ones, wouldn't be better to define these variables there?

Reply:
Currently they are only for backup restore, but I can't say for sure it's only there. We really want to have generic way of detecting flake and the problem may be in the future somewhere else where we want also to use this technique.